### PR TITLE
[cucumber] Add missing type in order to allow number as input parameter and tags on hooks + alignment

### PR DESCRIPTION
**Pull request by manu7691**
- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).

### CODE REVIEW

File: types/cucumber/index.d.ts [hunk 1]
```diff
@@ -62,14 +62,15 @@ declare namespace cucumber {
...
+	timeout?: number;
+	tags?: any;    
+    }
 
     export interface Hooks {
         Before(code: HookCode): void;
         Before(options: HookOptions, code: HookCode): void;
-		    After(code: HookCode): void;
-		    After(options: HookOptions, code: HookCode): void;
+	After(code: HookCode): void;
```
Comment (sandersn) [inline types/cucumber/index.d.ts]: Indentation is still wrong here.

Comment (manu7691) [inline types/cucumber/index.d.ts]: If you open the link, you will see it is proper set https://github.com/manu7691/DefinitelyTyped/blob/1f55084159d14af7cf93cff2e758714d0ff2b7cf/types/cucumber/index.d.ts

Comment (sandersn) [inline types/cucumber/index.d.ts]: Ugh, tabs. OK, makes sense.

Comment (manu7691) [inline types/cucumber/index.d.ts]: Anyway i reupdated the code based on IDE auto-alignment 👍

**Pull request merged**